docroot/themes/custom/uids_base/uids_base.theme

```diff
@@ -1094,18 +1094,18 @@ function uids_base_preprocess_block__inline_block__uiowa_collection(&$variables)
   $items = Element::children($variables['elements']['content']['field_uiowa_collection_items']);
   $variables['aria_owns'] = [];
   foreach ($items as $delta) {
+    $unique_id = $accordion_item_id . '-' . $delta;
+
+    // Set the unique ID to a stored value if it exists.
     $paragraph = $variables['elements']['content']['field_uiowa_collection_items'][$delta]['#paragraph'];
     if ($paragraph->hasField('field_unique_id') && !$paragraph->get('field_unique_id')->isEmpty()) {
-      // Use the unique ID if it exists.
       $unique_id = $paragraph->get('field_unique_id')->value;
-      $variables['aria_owns'][] = $unique_id;
-    }
-    else {
-      // Fall back to the delta-based ID.
-      $variables['aria_owns'][] = $accordion_item_id . '-' . $delta;
     }
+    $variables['aria_owns'][] = $unique_id;
   }
-
+  // Convert this to a comma-separated string as that is how it will be printed
+  // in the template.
+  $variables['aria_owns'] = implode(', ', $variables['aria_owns']);
```

Comment:
```suggestion
  $variables['aria_owns'] = implode(' ', $variables['aria_owns']);
```

Should this be space-separated instead of comma? The template was previously constructing it using `aria-owns="{{ _context['aria_owns']|join(', ') }}` which this matches, but the documentation I'm seeing has it as space-separated (eg https://www.w3.org/TR/wai-aria/states_and_properties#aria-owns). Not sure if there was a reason we'd had it commas previously.

Reply:
Good catch! Did you happen to check if Lighthouse flagged it with spaces vs commas? If it doesn't I'm good with making this change.

Reply:
I'm not seeing any difference with Lighthouse. With the Siteimprove extension, it flags the `aria-owns` on a `tablist` as incompatible with the commas but not with the spaces...which is good? According to the w3.org specs, tablist does support `aria-owns`, so maybe this is SI mislabeling what it's complaining about.